docs/docs/reference/unused-terms.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: doc-page
+title: "Unused Parameters"
+---
+
+Why unused parameters?
+----------------------
+The following examples shows an implementation of a simple state machine which can be in a state `On` or `Off`.
+The machine can change state from `Off` to `On` with `turnedOn` only if it is currently `Off`. This last constraint is
+captured with the `IsOff[S]` implicit evidence which only exists for `IsOff[Off]`.
+For example, not allowing calling `turnedOn` on in an `On` state as we would require an evidence of type `IsOff[On]` that will not be found.
+
+```scala
+sealed trait State
```

Comment:
Let's try to formulate this more directly. The fist example in a section should always highlight the feature we are introducing. Then, explain what the effect of the feature is. Writing code without it to motivate is too long-winded for the format we follow in these pages.